tests/App/config.yml

```diff
@@ -25,15 +25,15 @@ monolog:
             handler:      nested
         nested:
             type:  stream
-            path:  %kernel.logs_dir%/%kernel.environment%.log
+            path:  "%kernel.logs_dir%/%kernel.environment%.log"
```

Comment:
It catches this stuff automatically too?? Great!! :+1:

Reply:
@Jean85 No, I did it my self for this one. Yamllint is just a linter, not a fixer.

See: https://flintci.io/docs#yamllint

Reply:
But if you meant about linting, yes it catch it! :wink: